src/main/java/com/laa/model/crime/Court.java

```diff
@@ -0,0 +1,7 @@
+package com.laa.model.crime;
+
+public enum Court {
```

Comment:
Rename this using the enum convention CourtType instead of Court

Reply:
done

Reply:
done